NUService.js

```diff
@@ -156,6 +156,22 @@ export default class NUService extends NUObject {
         return url;
     }
 
+    buildBulkURL(entities, RESTResourceName, parentEntity) {
+        if (!entities || !entities[0]) {
```

Comment:
Lets force check if isAnArray and length > 0

Reply:
done